config/babel.prod.js

```diff
@@ -9,8 +9,7 @@
 
 module.exports = {
   presets: [
-    'babel-preset-es2015',
```

Comment:
Loose mode generates code that is simpler, but less strictly compliant to spec. Compare output:
- [es2015-loose](http://babeljs.io/repl/#?evaluate=false&lineWrap=false&presets=es2015-loose&experimental=true&loose=true&spec=false&code=import%20bar%20from%20'.%2Fbar'%3B%0A%0Aclass%20A%20%7B%0A%20%20foo()%20%7B%0A%20%20%20%20bar(this.c)%3B%0A%20%20%7D%0A%7D%0A%0Aexport%20default%20A%3B&playground=false)
- [es2015](http://babeljs.io/repl/#?evaluate=false&lineWrap=false&presets=es2015&experimental=true&loose=true&spec=false&code=import%20bar%20from%20'.%2Fbar'%3B%0A%0Aclass%20A%20%7B%0A%20%20foo()%20%7B%0A%20%20%20%20bar(this.c)%3B%0A%20%20%7D%0A%7D%0A%0Aexport%20default%20A%3B&playground=false)

I'm not aware of any reason to not at least run your production build in loose mode. I'm also not aware of anybody who deliberately uses non-loose mode.